index.d.ts

```diff
@@ -4,6 +4,7 @@ import { VNode } from "hyperapp";
 interface LinkProps {
   to: string;
   location?: Location;
+  class?: string;
```

Comment:
maybe `[attributeName]: any` ?

Reply:
I don't need it so didn't implement it. I don't know a way to be exhaustive here.